standard_init_linux.go

```diff
@@ -77,6 +78,9 @@ func (l *linuxStandardInit) Init() error {
 	if err != nil {
 		return err
 	}
+	if err := seccomp.InitSeccomp(l.config.Config.SeccompConfig); err != nil {
```

Comment:
I don't know, but I don't like idea that we just silently skip this step even if `l.config.Config.SeccompConfig` is not nil.